_includes/nav.html

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{{ '/'  | prepend: site.baseurl }}"><img width="20" src="{{ '/assets/img/scala-js-logo.svg' | prepend: site.baseurl }}"> {{ site.title }}</a>
+      <a class="navbar-brand" href="{{ '/'  | prepend: site.baseurl }}"><img width="120" alt="Scala.js" src="{{ '/assets/img/scala-js-site-logo.svg' | prepend: site.baseurl }}"></a>
```

Comment:
This was to get rid of a font used solely for the site logo. Also alignment and sizing of the icon + text is difficult to control for different devices (especially if people zoom in/out), so a single logo image is better.